HACKING.md

```diff
@@ -39,10 +39,11 @@ separated by an underscore. For example, `messages_gherkin-use-protobuf`.
 ### Local Testing / Development checks
 
 Cucumber runs with a highly coupled system of tests. Because cucumber is a mono-repo, testing these is best done
-in a docker container, due to needing a large amount of pre-requisites and often setting them up can be tricky.
+in a docker container, due to needing a large amount of pre-requisites. The setting up of these often can be a tricky
+task, so we use Docker to help us with this.
 
-To start with, we need to ensure we have Docker installed. This is usually fairly easy, and a comprehensive set
-of setup instructions are available [HERE](https://docs.docker.com/install/) for most OS'
+To start with, we need to ensure we have Docker installed. This is fairly straightforward, and a comprehensive set
```

Comment:
Let's not use words like "easy", "simple", "straightforward" in any of the documentation.  (see also: https://github.com/cucumber/docs.cucumber.io/issues/389)